tools/seistech_utils/seistech_utils/src/utils.py

```diff
@@ -2,12 +2,20 @@
 from pathlib import Path
 from typing import Tuple, Union, Sequence, List
 
-from importlib.metadata import version
-
-
 def get_package_version(package_name: str) -> Tuple[str, str]:
-    version_number = version(package_name)
+    try:
+        # Available for python version >= 3.8
+        from importlib.metadata import version
+    except ImportError:
+        try:
+            # Third party package
+            # https://pypi.org/project/importlib-metadata/
+            from importlib_metadata import version
+        except ImportError:
+            print(f"Please install the importlib-metadata package "
+                  f"or switch to a python version >= 3.8")
```

Comment:
This feels messy. But looks like the best way to do it.

Reply:
It is, I just added it for running on Maui, but since I gave up on that it probably makes sense to just remove it.